zap/src/main/java/org/zaproxy/zap/GuiBootstrap.java

```diff
@@ -347,9 +351,7 @@ private void setupLookAndFeel() {
             return;
         }
 
-        if (Constant.isMacOsX()) {
-            OsXGui.setup();
-        } else if (setLookAndFeel(getLookAndFeelClassname("Nimbus"))) {
```

Comment:
This else if really confused me here. 
What is the Nimbus Look & Feel Class and why is it incompatible with MacOS?

Reply:
It's not about incompatibility but use the most appropriate (prettier?) look and feel by default when running on macOS.

Reply:
Ah ok. But this else if would only run for **non** macOS Systems, right?

So all non macOS System use Nimbus and macOS by default uses the last line (`setLookAndFeel(UIManager.getSystemLookAndFeelClassName());`) of the method.

Reply:
That's correct.

Reply:
Ok understood. Thank you! 👏
Will push a small change to make sure that behavior is kept.

Reply:
Just to be clear:

> So all non macOS System use Nimbus

They default to Nimbus, but can still use other L&F.

Reply:
> Will push a small change to make sure that behavior is kept.

Thanks. The if moved above should be done after `lookAndFeelSet = true;`, it's not necessary to do that more than once.

Reply:
Okay applied they changes 👍